### PR TITLE
fix: update backstop withdraw fn and clean up rendering loop

### PR DESCRIPTION
**Pull request by mootz12**
Preview deployment available at: https://dev.blend-frontend.pages.dev/

### CODE REVIEW

File: src/pages/backstop.tsx [hunk 1]
```diff
@@ -126,17 +124,16 @@ const Backstop: NextPage = () => {
...
-      console.error('error on claim fn ');
+      console.error('Unable to estimate LP token mint amounts');
       setLoadingEstimate(false);
       setAvailableToMint('0');
     }
   }
 
   useEffect(() => {
-    if (balancesByAddress.get(backstopData?.config.usdcTkn ?? '') === undefined) {
-      loadUserData(walletAddress);
```
Comment (carlosdimatteo) [inline src/pages/backstop.tsx:137]: had added this to load usdc and blnd balances in case they're not there on page load

**Review by carlosdimatteo — DISMISSED** [review]
LGTM but left a comment around token balance loading

File: src/pages/backstop.tsx [hunk 2]
```diff
@@ -126,17 +124,16 @@ const Backstop: NextPage = () => {
...
+    if (balancesByAddress.get(backstopData?.config.usdcTkn ?? '') !== undefined) {
```
Comment (Ryang-21) [inline src/pages/backstop.tsx:134]: probably want to check if user has blnd to

**Review by Ryang-21 — CHANGES_REQUESTED** [review]
requested a small change but otherwise looks good

File: src/pages/backstop.tsx (hunk 1 above)
Comment (mootz12) [reply src/pages/backstop.tsx:137]: they should be, and this caused a situation where if you were on this page and the request failed for any reason, (like a rate limit), we begun spamming requests to the endpoint, causing long blocking times.

On normal usage, this was almost always loaded for me, so I think we should remove the potential for looping requests, and address this a different way if we find a way where it is still an issue.

**Review by Ryang-21 — APPROVED** [review]
lgtm

**Pull request merged**